libs/filamat/src/shaders/ShaderGenerator.cpp

```diff
@@ -63,6 +63,8 @@ void ShaderGenerator::generateSurfaceMaterialVariantDefines(io::sstream& out,
             filament::Variant::isVSMVariant(variant));
     CodeGenerator::generateDefine(out, "VARIANT_HAS_STEREO",
             hasStereo(variant, featureLevel));
+    CodeGenerator::generateDefine(out, "VARIANT_DEPTH",
```

Comment:
do we need to bump the material version?

Reply:
I don't think so, since this is backwards-compatible. Clients that want the new feature will need to re-build materials.

Reply:
I see, but maybe it'll be good to still add a line in the release note.  In case someone want's to depend on it and like to know which version of filament/matc would have it.